homework.md

```diff
@@ -4,77 +4,70 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is used to style websites. 
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
-}
-```
+n#id h1
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used make websites interactive
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to colllaborate on a project at the same time they can put it on Github, to track changes, leave comments and create a whole new project.
 
 ## Question 9
 
 What happens when you `fork` a repository?
-
-You delete it
+A copy of the repository
```

Comment:
Would be good to be a bit more specific here. Where does it copy to? From where?

Reply:
It copies the code to my profile repository page on github.com

Reply:
Can you do the change directly in the file and commit, pushed as usual. So that we can see the change. Thank you